Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -696,7 +711,7 @@
   - TOOLS
   - TOYS
   - TUBES
-  - VEHICLES
```

Comment:
vehicles do exist, shuttles exist

Reply:
well okay. but then it should be shuttles not vehicles. think of what a borg would do when they get that as a law.

Reply:
Added shuttles to appease you :trollface: